### ISSUE TITLE
Spike: Including ACH in the UPE

### ISSUE THREAD

**Issue by jessepearson** [Automattic/woocommerce-payments#3445]
Trying to determine what it would take to get ACH in the UPE.

**Comment by jessepearson** [comment]
I started looking into this last week and found the documentation here:
* https://stripe.com/docs/ach
* https://stripe.com/docs/sources/ach-credit-transfer

On Monday, I touched base with @ismaeldcom about it, and he found the same documents. @tpaksu also jumped in on the conversation here: p1638191640330000-slack-C01B8KNUYSW

We all found that it didn't seem like ACH *could* be included in the UPE, as the documentation didn't seem to read that way. This led me to reach out to Stripe directly about it: p1638197967188200-slack-C9976E5MJ

Stripe informed me it is available on the Dev account and directed me to other documentation here: https://stripe.com/docs/payments/us-bank-account/accept-a-payment . The new documentation can only be seen when logged into the Dev account, and so I moved forward with trying to quickly mock it up. This led me to further errors, which I communicated to Stripe in the above thread. 

Stripe stated further setup was needed on their end, and they would communicate back once that was done.
